src/components/ImageFigure/__tests__/image-figure.test.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import React from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
Should this be switched to the package you just included "@testing-library/jest-dom"?